.github/WHAT_IS_MAKER.md

```diff
@@ -0,0 +1,63 @@
+# What is Maker?
+
+Most people contributing to and using Maker will be familiar with [Orbit](https://github.com/square/orbit) and will likely have a lot of wrong ideas about Maker's goals by assuming they're the same as Orbit's. This is incorrect. Maker is not Orbit. Their key high-level differences in a table:
+
+| | Orbit | Maker |
```

Comment:
This is awesome. Can you give examples of high-level and mid-levels components in parens on the API Design row.

Reply:
Kinda hard to clearly explain it in some inline parens within the table, I may try to go into an more in-depth explanation later somewhere else in this doc.